File: Javascript/1012.js
var input = require('fs').readFileSync('stdin', 'utf8');

var [A, B, C] = input.split(" ").map(item => parseFloat(item));
var PI = 3.14159

var areaTriangulo = (A * C) / 2;
var areaCirculo = PI * Math.pow(C, 2);
var areaTrapezio = (A + B) *  C / 2;
var areaQuadrado = Math.pow(B, 2);
var areaRetangulo = A * B

console.log("TRIANGULO: " + areaTriangulo.toFixed(3))
console.log("CIRCULO: " + areaCirculo.toFixed(3))
console.log("TRAPEZIO: " + areaTrapezio.toFixed(3))
console.log("QUADRADO: " + areaQuadrado.toFixed(3))
console.log("RETANGULO: " + areaRetangulo.toFixed(3))

File: Java/1012.java
import java.util.Scanner;
import java.lang.Math;

public class Main {

  public static void main(String[] args)  {
    Scanner scanner = new Scanner(System.in);
    
    double pi = 3.14159;
    double A = scanner.nextDouble();
    double B = scanner.nextDouble();
    double C = scanner.nextDouble();

    double areaTriangulo = (A*C)/2;
    double areaCirculo = pi * Math.pow(C,2);
    double areaTrapezio = ((A+B) * C)/2;
    double areaQuadrado = Math.pow(B,2);
    double areaRetangulo = A * B;

    String areaTrianguloFormatada = String.format("%.3f", areaTriangulo);
    String areaCirculoFormatada = String.format("%.3f", areaCirculo);
    String areaTrapezioFormatada = String.format("%.3f", areaTrapezio);
    String areaQuadradoFormatada = String.format("%.3f", areaQuadrado);
    String areaRetanguloFormatada = String.format("%.3f", areaRetangulo);

    System.out.println("TRIANGULO: " + areaTrianguloFormatada);
    System.out.println("CIRCULO: " + areaCirculoFormatada);
    System.out.println("TRAPEZIO: " + areaTrapezioFormatada);
    System.out.println("QUADRADO: " + areaQuadradoFormatada);
    System.out.println("RETANGULO: " + areaRetanguloFormatada);

  }
}